packages/governance/test/swingsetTests/contractGovernor/governor.js

```diff
@@ -0,0 +1,41 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+import { E } from '@agoric/eventual-send';
+import { setupGovernance } from './governParam';
+
+/** @type {ContractStartFn} */
+const start = async zcf => {
+  const zoe = zcf.getZoeService();
+
+  const startGovernedInstance = async (
```

Comment:
I was thinking that unlike packages/governance/src/contractGovernor.js, this would only govern the one contract instance.